javascript/javascript1/week4/js1-week4-homework/index.js

```diff
@@ -0,0 +1,92 @@
+const name = [];
+const toDo = [];
+function getReply(command) {
+  const lowerCase = command.toLowerCase();
+  //console.log(lowerCase)
+
+  let words = lowerCase.split(" ");
+  let myName = words[words.length - 1];
+  name.push(myName);
+  //console.log(name)
```

Comment:
I think you want this to be part of the first `if` block below here. And the `name` should likely be `let name = '';` instead since it should only store one name at a time. 
Currently each time a command is run, the last word in that command is pushed to the name array.